source/MaterialXGenShader/ShaderStage.cpp

```diff
@@ -354,4 +354,43 @@ void ShaderStage::addFunctionDefinition(const ShaderNode& node, GenContext& cont
     }
 }
 
+string ShaderStage::getUniformValues() const {
```

Comment:
Hopefully we can get rid of writing JSON strings manually, once MaterialX supports JSON natively 😅

Reply:
@bernardkwok is working on a new api so that we can get rid of the json entirely and use something like a key value map instead. So this is just a workaround until that api is available